pkgs/tools/wayland/gtklock/default.nix

```
@@ -0,0 +1,48 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, pkgs
+, pam
+, scdoc
+, gtk3
+, pkg-config
+, gtk-layer-shell
+, wayland
+}:
+stdenv.mkDerivation rec {
+  pname = "gtklock";
+  version = "1.3.4";
+
+  src = fetchFromGitHub {
+    owner = "jovanlanik";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "0m5z863ihaxy4460c06m3g1y01ivgj4hadq302pcfdmh2ipqcf1v";
+  };
+
+  buildInputs = [
+    scdoc
+  ];
+  nativeBuildInputs = [
+    wayland
+    gtk3
+    pkg-config
+    pam
+    gtk-layer-shell
+  ];
```

```suggestion
  strictDeps = true;

  nativeBuildInputs = [
    scdoc
    pkg-config
  ];

  buildInputs = [
    wayland
    gtk3
    pam
    gtk-layer-shell
  ];
```

strictDeps enforces the extra strictness for cross compiling also when not cross compiling. If the package does not build with it then the dependencies are not quite correct.